modules/my_health/app/controllers/my_health/v1/prescriptions_controller.rb

```diff
@@ -58,10 +57,9 @@ def refill_prescriptions
       def list_refillable_prescriptions
         resource = collection_resource
         resource.data = filter_data_by_refill_and_renew(resource.data)
-        render json: resource.data,
-               serializer: CollectionSerializer,
-               each_serializer: PrescriptionDetailsSerializer,
-               meta: resource.metadata
+
+        options = { meta: resource.metadata, is_collection: true }
```

Comment:
I haven't used/seen is_collection yet, what is that doing?

Reply:
`jsonapi-serializer` attempts to determine automatically if the object (in this case resource) is a collection or a single object. 

However, I don't think it's necessary in this case, because resource.data is an array. I'll remove it because it's redundant.